### PR TITLE
Add extra params

### PR DESCRIPTION
**Pull request by yyyogev**
@imnotashrimp please review the sections I tagged you at

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ java -javaagent:~/jmx2logzio/target/jmx2logzio-1.0.1.jar=LOGZIO_TOKEN=<ACCOUNT-T
 | **POLLING_INTERVAL_IN_SEC** | Metrics polling interval, in seconds. <br /> **Default**: `30` |
 | **WHITE_LIST_REGEX** | Only metrics matching this regex will be sent. <br /> **Default**: `.*` (match everything) |
 | **BLACK_LIST_REGEX** | Metrics matching this regex will not be sent. <br /> **Default**: `$a` (match nothing) |
+| **EXTRA_DIMENSIONS** | A list of key-values separated by ':' that will be added to the dimensions of the collected metrics. <br /> Example: EXTRA_DIMENSIONS={origin=local:env=java} |
```
Comment (yyyogev) [inline README.md]: @imnotashrimp

File: README.md [hunk 2]
```diff
@@ -124,6 +125,7 @@ Open jmx2logzio/src/main/resources/application.config in a text editor, and set
 | **service.poller.black-list-regex** | Metrics matching this regex will not be sent. <br /> **Default**: `$a` (match nothing) |
 | **service.poller.jolokia.jolokiaFullUrl** | URL of the remote Jolokia agent you're forwarding metrics to. |
 | **service.poller.metrics-polling-interval-in-seconds** | Metrics polling interval, in seconds. <br /> **Default**: `30` |
+| **extra-dimensions** | A dictionary of key-values that will be added to the dimensions of the collected metrics |
```
Comment (yyyogev) [inline README.md]: @imnotashrimp

Comment (idohalevi) [reply README.md]: add: `Example: extra-dimensions={origin=local:env=java}`

File: src/main/java/io/logz/jmx2logzio/Jmx2Logzio.java [hunk 3]
```diff
@@ -30,6 +32,8 @@ public Jmx2Logzio(Jmx2LogzioConfiguration conf) {
         this.taskScheduler = newSingleThreadScheduledExecutor();
 
         this.client = conf.getMetricClientType() == JOLOKIA ? new JolokiaClient(conf.getJolokiaFullUrl()) : new JavaAgentClient();
+        List<Dimension> extraDimensions =conf.getExtraDimensions();
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/Jmx2Logzio.java]: add space

File: src/main/java/io/logz/jmx2logzio/configuration/Jmx2LogzioConfiguration.java [hunk 4]
```diff
@@ -58,6 +64,14 @@ public Jmx2LogzioConfiguration(Config config) throws IllegalConfiguration {
...
         setListenerURL(config);
 
+        extraDimensions = new ArrayList<>();
+        if (config.hasPath(Jmx2LogzioJavaAgent.EXTRA_DIMENSIONS)) {
+            if (metricClientType == MetricClientType.MBEAN_PLATFORM) {
+                extraDimensions = parseExtraDimensions(config.getString(Jmx2LogzioJavaAgent.EXTRA_DIMENSIONS));
+            } else {
+                extraDimensions = parseExtraDimensions(config.getConfig(Jmx2LogzioJavaAgent.EXTRA_DIMENSIONS));
+            }
+        }
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/configuration/Jmx2LogzioConfiguration.java]: Why we need the second if?

**Pull request merged**